Source/WebKit/Shared/cg/ShareableBitmapCG.mm

```diff
@@ -115,7 +115,20 @@ RefPtr<ShareableBitmap> ShareableBitmap::createFromImagePixels(NativeImage& imag
     if (colorSpace != DestinationColorSpace::SRGB())
         return nullptr;
 
-    auto pixels = adoptCF(CGDataProviderCopyData(CGImageGetDataProvider(image.platformImage().get())));
+    auto configuration = ShareableBitmapConfiguration(image);
+
+    RetainPtr<CFDataRef> pixels;
+    @try {
+        pixels = adoptCF(CGDataProviderCopyData(CGImageGetDataProvider(image.platformImage().get())));
+    } @catch (id exception) {
+        LOG_WITH_STREAM(Images, stream
```

Comment:
Log the image URL if you can.

Reply:
I can't see a way to get from NativeImage to the originating URL.